zenoh/src/net/routing/dispatcher/pubsub.rs

```diff
@@ -124,7 +124,7 @@ pub(crate) fn undeclare_subscription(
     node_id: NodeId,
     send_declare: &mut SendDeclare,
 ) {
-    tracing::debug!("Undeclare subscription {}", face);
+    tracing::trace!("Undeclare subscription {}", face);
```

Comment:
I would just remove this trace. It's not coherent with the rest of the code and contains too few info. It's probably a leftover from a debug session.